src/strategies/layers/connector/morpho/MorphoConnector.sol

```diff
@@ -51,17 +51,17 @@ abstract contract MorphoConnector is ERC4626Connector {
     for (uint256 i = 0; i < rewardTokens.length; ++i) {
       address rewardToken = rewardTokens[i];
       Rewards memory rewardsMem = rewards[rewardToken];
+      if (rewardsMem.lastUpdated == 0) {
```

Comment:
We moved this outside of the `if (unaccounted > 0) {` if because it made sense to decouple token registration from whether there unaccounted balance. It should make things easier for admins